server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -720,6 +754,11 @@ pub fn router() -> ApiRouter<AppState> {
             get(get_messageattempt),
             openapi_tag("Message Attempt"),
         )
+        .api_route_with(
+            "/app/:app_id/msg/:msg_id/attempt/:attempt_id/response/",
```

Comment:
```suggestion
            "/app/:app_id/msg/:msg_id/attempt/:attempt_id/content/",
```
Probably let's make it consistent with msg? I'm on the fence here to be honest, but I think calling both `content` or maybe we should change both to `body`?

Reply:
I'm leaning toward `content`.

Reply:
Yeah, you're right. Especially because I just remembered we have `with_content` as a query param in various places.